src/com/itacademy/eshop/shop/ShoppingCart.java

```diff
@@ -1,11 +1,13 @@
 package com.itacademy.eshop.shop;
 
 import com.itacademy.eshop.product.Product;
+import com.itacademy.eshop.product.Review;
 
 import java.util.ArrayList;
 
 public class ShoppingCart {
     ArrayList<Product> products;
```

Comment:
You have to add access modifier. By default it will be package private. Your class variables should be private and only accessed via getters